apps/web/src/utils/experienceUtils.test.ts

```diff
@@ -1,8 +1,9 @@
 /**
- * @jest-environment jsdom
+ * @vi-environment jsdom
```

Comment:
All the other tests remove this comment.  This looks like a find-and-replace mistake?

Reply:
Oh good catch! Removed in [remove comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14886/commits/5f0daf761be94c1cb4fe03d535b99734344ca0fe)